### ISSUE TITLE
[Feature-Request] A new subfolder should inherit the settings of the parent folder

### ISSUE THREAD

**Issue by Sogolumbo** [k9mail/k-9#4852]
A newly created subfolder should inherit the settings of their parent folder.

Example:
I create a new subfolder (Inbox/mailing-list) in my inbox for emails from a mailing list.
Using a thunderbird filter all mails with "[mailing-list]" in the subject are moved to the new folder.
Now I'd still like those emails to pop up in the common inbox of all accounts.

Current behaviour:
After refreshing the folder list the folder will be created with default settings and new emails in that folder won't appear in the common inbox.
The current behaviour always requires some time to configure new subfolders.

**Comment by cketti** [comment]
K-9 Mail currently doesn't know about the folder hierarchy. So this feature depends on issue #630.

Overall, I'd say this is a very low priority feature.
